app/services/search/search_builder.rb

```diff
@@ -0,0 +1,76 @@
+class Search::SearchBuilder
+  DEFAULT_HITS_PER_PAGE = 10
+
+  attr_reader :hits_per_page, :keyword, :location_search, :page, :params_hash,
+              :point_coordinates, :search_filters, :search_replica, :sort_by, :stats, :vacancies,
+              :wider_radius_suggestions
+
+  def initialize(form_hash)
+    @params_hash = form_hash
+    @keyword = @params_hash[:keyword]
+    @hits_per_page = @params_hash[:per_page] || DEFAULT_HITS_PER_PAGE
+    @page = @params_hash[:page]
+
+    initialize_search
```

Comment:
Did Rubocop ask you to package these things up and refactor them out of the main `initialize` method? I ask because I think it's actually harder to read when
```ruby
build_location_search
build_search_filters
build_search_replica
call_algolia_search
```
are far away elsewhere in the file. I'd prefer to read those steps in the `initialize` at that level of granularity, because when I read the method `initialize_search` being called, I wonder 'now what does that mean' and go searching.

What a lot of words for a very minor quibble. Take it or leave it.

Reply:
Yeah that makes sense, will chuck them in and let rubocop squeal